roles/load_aware_undeploy_trimaran/tasks/main.yml

```
@@ -0,0 +1,21 @@
+- name: Delete Trimaran resources
+  shell: |
+    oc delete -f {{ trimaran_setup_config }} 2>&1
+  register: delete_trimaran
+  failed_when: delete_trimaran.rc != 0 and 'namespaces "trimaran" not found' not in delete_trimaran.stdout
+
+- name: Delete user monitoring configmap
+  shell: |
+    oc delete -f {{ user_applications_monitor_config }} 2>&1
+  register: delete_monitoring
+  failed_when: delete_monitoring.rc != 0 and 'configmaps "cluster-monitoring-config" not found' not in delete_monitoring.stdout
```

I was going to suggest this:
```suggestion
  command:
    oc delete -f {{ user_applications_monitor_config }} --ignore-not-found
```
but I prefer that we don't touch this config map, we don't know who created it ... 
(and we don't really mind if user workload monitoring remains enabled)

---

I can go ahead and just remove this, but to deploy Trimaran we do have to first deploy this config map.
https://github.com/openshift-psap/ci-artifacts/blob/4b7be770286039f0bc03c254d31c43e6d66d3474/roles/load_aware_deploy_trimaran/tasks/main.yml#LL1C1-L3C55

As you mentioned it's probably not a big deal if it _does_ stay configured, but we are actually the ones that create it :)

---

yes, I prefer that we don't touch this resource (=we just create it in the deployment stage)
rational is:
- if nobody enabled it before, we enable it and let it enabled
- if somebody enabled it before, we cannot (easily) know it. So just in case, we let it enabled

---

just FYI, the reason why it's not enabled by default is that cluster admins don't want that _any_ payload installed by the cluster users to generate metrics that are stored in Prometheus.
In a controlled environment where we trust the workload that we deploy, it's safe to let this feature enabled

---

Makes sense!